controls/SV-204393.rb

```diff
@@ -1,35 +1,23 @@
 control 'SV-204393' do
-  title 'The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent
-    Banner before granting local or remote access to the system via a graphical user logon.'
-  desc 'Display of a standardized and approved use notification before granting access to the operating system
+  title "The Red Hat Enterprise Linux operating system must display the Standard Mandatory #{input('org_name')} Notice and Consent
+    Banner before granting local or remote access to the system via a graphical user logon."
```

Comment:
After the input value for `org_name` is updated with the keys, this will need to be updated to select the relevant keys